src/Umbraco.Tests/Testing/TestOptionAttributeBase.cs

```diff
@@ -35,11 +35,11 @@ public static TOptions GetTestOptions<TOptions>()
             if (type == null)
             {
                 type = ScanAssemblies
-                    .Select(x => Type.GetType(typeName + ", " + x, false))
+                    .Select(assembly => assembly.GetType(typeName, false))
                     .FirstOrDefault(x => x != null);
                 if (type == null)
                 { 
-                    throw new PanicException($"Could not resolve the type from type name {typeName}"); // makes no sense
+                    throw new PanicException($"Could not resolve the running test fixture from type name {typeName}"); // makes no sense
```

Comment:
Took the liberty of making this exception text a bit more specific as well.  
I'm quite confident this attribute shouldn't be on non-fixtures.